spectractor/extractor/spectrum.py

```diff
@@ -503,6 +511,116 @@ def plot_spectrogram(self, ax=None, scale="lin", title="", units="Image units",
         if parameters.PdfPages:  # pragma: no cover
             parameters.PdfPages.savefig()
 
+    def plot_spectrum_summary(self, xlim=None, figsize=(12, 12), saveAs=''):
+        """Plot spectrum with emission and absorption lines.
+
+        Parameters
+        ----------
+        ax: Axes, optional
+            Axes instance (default: None).
+        label: str
+            Label for the legend (default: '').
+        xlim: list, optional
+            List of minimum and maximum abscisses (default: None)
+        live_fit: bool, optional
+            If True the spectrum is plotted in live during the fitting procedures
+            (default: False).
+        force_lines: bool
+            Force the over plot of vertical lines for atomic lines if set to True (default: False).
```

Comment:
A few of these docstrings don't match the function definition.

Reply:
Oh, and you use `snake_case` so you might want to rename the variable I added (`saveAs`) - sorry about that.

Reply:
OK